configure.ac

```diff
@@ -86,16 +91,28 @@ PKG_CHECK_MODULES([HARFBUZZ], [harfbuzz >= 1.2.3], [
 libpng=false
 AS_IF([test "x$enable_test" = xyes || test "x$enable_compare" = xyes], [
     PKG_CHECK_MODULES([LIBPNG], [libpng >= 1.2.0], [
+        # Only used for test programs, must not be used for distribution
         CFLAGS="$CFLAGS $LIBPNG_CFLAGS"
         AC_DEFINE(CONFIG_LIBPNG, 1, [found libpng via pkg-config])
         libpng=true
     ])
 ])
 
+AS_IF([test "x$enable_unibreak" != xno], [
+    PKG_CHECK_MODULES([LIBUNIBREAK], [libunibreak >= 1.1], [
+        CFLAGS="$CFLAGS $LIBUNIBREAK_CFLAGS"
+        LIBS="$LIBS $LIBUNIBREAK_LIBS"
+        AC_DEFINE(CONFIG_UNIBREAK, 1, [found libunibreak via pkg-config])
```

Comment:
Don’t we need to add libunibreak to `pkg_requires` as well? From a glance at libunibreak’s readme, it seems the situation is messy about dynamic vs static builds but dynamic builds are possible and done by default on UNIXes?

Reply:
Yes, we should add it to `pkg_requires`. The first commit moving the `pkg_requires` definition to the pkg-config checks was done to make this simpler. … and yet I forgot to actually add it. Thanks for pointing this out, is now fixed.

To my understanding libunibreak has 3 build systems:
 - First, the preferred buildsystem is autotools + libtool. It can regenerate the Unicode data and build both static and dynamic libs. It works aon all platforms supported by libtool, so basically all unix-like systems and MS WIndows via Cygwin.
  - Then there's a manual Makefile requiring a gcc-syntax C-compiler. This can also regenerate Unicode data, but only build static libs *(as dynamic libs require all sort of system specific configuration)*. This works basically everywhere a gcc-syntax compatible C-compiler is available *(which includes Windows thanks to mingw)*
   - Lastly there's a Makefile using Micorosoft's VSC-compiler. It cannot regenerate Unicode data, only works in MS Windows and can only build static libs.
     Curiously this Makefile appears to have an unused variable `DEF_FILE` for a definition file *(this can be used with MSVC instead of putting `dllexport` stuff in public headers)*. If this Makefile could generate a def file from libtools sym file and use it, it would also be possible to do dynamic builds with MSVC.